<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listDialog</class>
 <widget class="QDialog" name="listDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>播放列表</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#listDialog
{ 
background-image: url(:/new/player of icon/image/壁纸.jpg);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>列表</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/player of icon/image/添加1.ico</normaloff>:/new/player of icon/image/添加1.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/player of icon/image/减少1.ico</normaloff>:/new/player of icon/image/减少1.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="scanButton">
         <property name="text">
          <string>scan</string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/player of icon/image/浏览.ico</normaloff>:/new/player of icon/image/浏览.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
